CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+## Developer Tool Prerequisites
+* [go](https://golang.org/doc/install) - The Go programming language
+* [dep](https://golang.github.io/dep/docs/installation.html) - Go dependency manager
+* [nvm](https://github.com/creationix/nvm#installation) - Node Version Manager (for installing NodeJS and NPM)
+
+## Performing a Fresh Checkout
+The following steps guide you through a fresh checkout
+
+```
+# Create a workspace
+cd ~/Projects/grpc-web  # or wherever you want your checkout to live
+export GOPATH=`pwd`
```

Comment:
I prefer `$()`, I think backtick notation is only used for compatibility, right?